lib/Target/LLVMIR/LLVMIRTranslation.cpp

```diff
@@ -86,6 +86,7 @@ static void amendLLVMFunc(llvm::Function *func, const NVVMMetadata &metadata,
   if (metadata.isKernel) {
     if (isROCM) {
       func->setCallingConv(llvm::CallingConv::AMDGPU_KERNEL);
+      func->addFnAttr("amdgpu-waves-per-eu", "3");
```

Comment:
Is it intentional change?

Reply:
Yes. This is for external clients to reproduce our best FA fwd perf. I forgot to mention it in the description.

Reply:
@binarman So can you add a blocker to this PR?

Reply:
Well, looks like this account can not do this :(

I have two accounts: @binarman (**not enrolled** into rocm organization) and @alefimov-amd (**enrolled** into rocm organization)

Recently I've lost access to MFA for alefimov-amd, so I can not use it to review PRs. I am in process of solving this problem.